package.json

```
@@ -1,42 +1,48 @@
 {
   "name": "lz-string",
+  "author": {
+    "name": "pieroxy",
+    "email": "pieroxy@pieroxy.net"
+  },
+  "contributors": [
+    {
+      "name": "Karnthis",
+      "email": "me@erinrivas.com"
+    }
+  ],
   "version": "1.5.0",
   "license": "MIT",
-  "filename": "lz-string.js",
   "description": "LZ-based compression algorithm",
   "homepage": "http://pieroxy.net/blog/pages/lz-string/index.html",
-  "keywords": [
-    "lz",
-    "compression",
-    "string"
-  ],
-  "main": "libs/lz-string.js",
-  "typings": "typings/lz-string.d.ts",
-  "bin": {
-    "lz-string": "bin/bin.js"
-  },
-  "scripts": {},
-  "dependencies": {},
-  "devDependencies": {},
   "repository": {
     "type": "git",
     "url": "https://github.com/pieroxy/lz-string.git"
   },
   "bugs": {
     "url": "https://github.com/pieroxy/lz-string/issues"
   },
-  "directories": {
-    "test": "tests"
+  "type": "module",
+  "keywords": [
+    "lz",
+    "compression",
+    "string"
+  ],
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "test": "vitest",
+    "test:coverage": "vitest run --coverage"
   },
-  "author": "pieroxy <pieroxy@pieroxy.net>",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/pieroxy/lz-string.git",
-    "basePath": "libs/",
-    "files": [
-      "lz-string.js",
-      "lz-string.min.js",
-      "base64-string.js"
-    ]
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^25.0.0",
+    "@rollup/plugin-typescript": "^11.1.1",
+    "prettier": "^2.8.8",
```

One thing I don't like with prettier is how it ignores coding standards for "operators at the start of the line" - would you have any objections to the fork which actually does it correctly? (Pretty much the biggest issue on their repo) -

```suggestion
    "@btmills/prettier": "2.8.8",
    "prettier": "npm:@btmills/prettier@~2.8.8",
```

Related - remove the `^` prefix on versions (ie, lock them down for reproducibility), and add a `.npmrc` containing -

```
tag-version-prefix=""
save-prefix=""
```

---

switched it out, I like that better. not going to run prettier until we are done because it touches nearly everything.